src/priorityQueue.js

```diff
@@ -0,0 +1,72 @@
+export class Element {
+  constructor(value, priorityInt) {
+    this.data = value
+    this.priority = priorityInt
+  }
+}
+
+export default class PriorityQueue {
+  constructor() {
+    this.elements = []
+    this.size = 0
+  }
+
+  enqueue(value, priorityInt) {
+    const element = new Element(value, priorityInt)
```

Comment:
Consider using the declaration inline, to avoid the use of the temporary variable:
```js
this.elements.push( new Element( value, priorityInt ) )
```
Also, I think it is sufficient to name the identifier `priority` - what if the type of the priority changes?  The use of the type in the identifier would then mean that the identifier was misleading, and would probably need to be changed.  Simply using an identifier that clearly communicates the intent of the variable is preferable to using the expected type name in the identifier.